src/call-service.js

```diff
@@ -89,7 +89,22 @@ const callService = ({
       throw new Error(`Service ${namespace || defaultNamespace()}.${service}.${handler} not found`);
     }
 
-    const [serviceToRun] = Instances;
```

Comment:
What if a service has multiple instances, and doesn't use the 'handler' key to find the specific one, which is the case for anything that's not a Lambda function...